Editor/MaterialPropertyNameRemapEditor.cs

```diff
@@ -1,19 +1,19 @@
-using System;
-using System.Collections;
 using System.Collections.Generic;
 using UnityEditor;
 using UnityEngine;
 using Object = UnityEngine.Object;
 
 namespace Unity.MaterialSwitch
 {
-
     [CustomEditor(typeof(MaterialPropertyNameMap))]
     public class MaterialPropertyNameRemapEditor : Editor
     {
         private Dictionary<int, MaterialPropertyNameMap> _nameMaps = new Dictionary<int, MaterialPropertyNameMap>();
 
-       
+        private MaterialPropertyReorderableList _propertyList;
+
+        private string _filterText;
```

Comment:
I am thinking that perhaps it's better to serialize this in `MaterialPropertyNameMap` asset so that we can reuse it after clicking another object ? 
What do you think ?

Reply:
We cannot, it uses a class from the Editor assembly.

Reply:
We can if we store the filterText in the runtime class, which is `MaterialPropertyNameMap`.
That being said, I think we don't have to do it for this PR.